src/components/NcButton/NcButton.vue

```diff
@@ -509,13 +509,10 @@ export default {
 	}
 
 	&__icon {
-		height: $clickable-area;
-		width: $clickable-area;
-		min-height: $clickable-area;
-		min-width: $clickable-area;
 		display: flex;
 		justify-content: center;
 		align-items: center;
+		margin: auto 8px auto 12px;
```

Comment:
This has to be reverted. It breaks the icon alignment.

Reply:
> This has to be reverted. It breaks the icon alignment.

Most obvious for the `Actions` component:
![Screenshot 2022-08-19 at 10-20-04 Nextcloud Vue Style Guide](https://user-images.githubusercontent.com/2496460/185576353-26c385ea-42dc-42fe-a046-ed707e3ac9c9.png)

Reply:
:disappointed: :+1: